docs/ref-bundle-stages.md

```diff
@@ -17,3 +17,40 @@ To demonstrate the life cycle of a Fleet bundle, we will use [multi-cluster/helm
 This diagram shows the different rendering stages a bundle goes through until deployment.
 
 ![Bundle Stages](/img/FleetBundleStages.svg)
+
+## Examining the Bundle Lifecycle With the CLI
+
+Several fleet CLI commands help with debugging bundles.
+
+### fleet apply
+
+[Apply](./cli/fleet-cli/fleet_apply.md) renders a folder with Kubernetes resources, such as a Helm chart, manifests, or kustomize folders, into a Fleet bundle resource.
+
+```
+git clone https://github.com/rancher/fleet-test-data
+cd fleet-test-data
+fleet apply -n fleet-local -o bundle.yaml testbundle simple-chart/
+```
+
+More information on how to create bundles with `fleet apply` can be found in the [section on bundles](https://fleet.rancher.io/bundle-add).
+
+### fleet target
+
+[Target](./cli/fleet-cli/fleet_target.md) reads a bundle from a file and works with a live cluster to print out the `bundledeployment` & `content` resource, which fleetcontroller would create. It takes a namespace as an argument, so it can look in that namespace for e.g. cluster resources. It can also dump the data structure which is used during "targeting", so decisions taken regarding labels and cluster names can be checked.
```

Comment:
nit: this feels a bit more readable to me:
```suggestion
[Target](./cli/fleet-cli/fleet_target.md) reads a bundle from a file, and works with a live cluster to print out the `bundledeployment` & `content` resources which fleetcontroller would create. It takes a namespace as an argument, so it can look in that namespace for e.g. cluster resources. It can also dump the data structure which is used during "targeting", so decisions taken regarding labels and cluster names can be checked.
```
Also, why quote the word _targeting_? Do we want to link from it to eg. [this page](https://fleet.rancher.io/gitrepo-targets#defining-targets), to clarify what is meant by _targeting_?